problems/iterating_objects/problem.txt

```diff
@@ -0,0 +1,57 @@
+Javascript has poor standard library. It lacks many functions available to programer in other script languages like ruby or python.
+We need to include additional libraries which implements those function.
+jQuery itself provides us with few utility functions.
+In this exercise we will explore two iteration methods, '$.each()' and '.each()'.
+First one is applied to arrays, it takes two arguments, array to iterate and iteration function.
+
+{{ code }}
+  var myArray = [1, 2, 3];
+  $.each(array, function(index, element) {
+    ....
+  });
+{{/ code }}
+
+It is alternative to standard for loop.
+
+{{ code }}
+  var myArray = [1, 2, 3];
+  $.each(array, function(index, element) {
+    ....
+  });
+{{/ code }}
+
+The second one, '.each()' will be more interesting for us. It operates on collection of elements and is used more often.
+It is called on jQuery objetct and takes one argument, function to call for each iteration with two arguments, index and element.
+
+{{ code }}
+  $('.myClass').each(function(index, element) {
```

Comment:
@pokorson I think it is bad example to iterate over elements to add a class (i see it is as a bad practice), because `jQuery` allows us to add that class to all elements without iteration.
I see that each useful when I want collect some data from `data-attributes`.